lib/editor/components/timetable/TimetableEditor.js

```diff
@@ -293,10 +312,10 @@ export default class TimetableEditor extends Component<Props, State> {
       return
     }
     showConfirmModal({
-      title: `Delete trips?`,
-      body: `Are you sure you want to permanently delete ${selectedCount} selected trip${selectedCount > 1 ? 's' : ''}?`,
+      title: this.messages('deleteTripsQuestion'),
+      body: this.messages(selectedCount > 1 ? 'deleteSelectedTrip' : 'deleteSelectedTrips').replace('%count%', selectedCount.toString()),
```

Comment:
the plural version appears to be reversed with the singular text.

Reply:
Wow (https://github.com/ibi-group/datatools-ui/pull/1032/commits/83fbd3dab67262df5a8a973160c230c86cb1bd6a)